src/gamekit.cs

```diff
@@ -1403,11 +1407,13 @@ interface GKAchievement : NSSecureCoding {
 #if MONOMAC
 		[Mac (10,10)]
 		[Export ("challengeComposeControllerWithMessage:players:completionHandler:")]
+		[Async (ResultTypeName = "GKChallengeComposeResult")]
 		NSViewController ChallengeComposeController ([NullAllowed] string message, GKPlayer [] players, [NullAllowed] GKChallengeComposeHandler completionHandler);
 #else
 		[NoWatch]
 		[iOS (8,0)]
 		[Export ("challengeComposeControllerWithMessage:players:completionHandler:")]
+		[Async (ResultTypeName = "GKChallengeComposeResult")]
```

Comment:
I do not think we can Async'ify this one according to [docs](https://developer.xamarin.com/guides/cross-platform/macios/binding/binding-types-reference/#AsyncAttribute):

> You can use the `[Async]` attribute on methods that **return void** and whose **last** argument is a callback.